<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Bhuvaneshs-MBP.attlocal.net" name="Test-class test" tests="2" failures="1" timestamp="5 Dec 2019 02:56:10 GMT" time="5.459" errors="0">
  <testcase name="testcase2" time="5.399" classname="test.TestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at test.TestCases.testcase2(TestCases.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testcase2 -->
  <testcase name="testcase1" time="5.445" classname="test.TestCases"/>
</testsuite> <!-- Test-class test -->
